datanode/gateway/graphql/schema.graphql

```diff
@@ -724,14 +724,13 @@ type RecurringTransfer {
 }
 
 enum DispatchMetric {
+  DISPATCH_METRIC_TAKER_FEES_PAID
 
-  MarketTradingValue
+  DISPATCH_METRIC_MAKER_FEES_RECEIVED
 
-  MakerFeesReceived
+  DISPATCH_METRIC_LP_FEES_RECEIVED
 
-  TakerFeesPaid
-
-  LPFeesReceived
```

Comment:
question (non-blocking): some of these have disappeared, i assume no longer needed?

Reply:
Incorrect, they are still all here?

Reply:
yep ignore me